web-app/src/app/app/jobs/components/jobs-table.tsx

```diff
@@ -29,6 +31,10 @@ export default function JobsTable({ jobs }: JobsTableProps) {
   return (
     <DataTable
       columns={columns}
+      rowOnClick={(row) => () => {
+        router.push(`/app/jobs/${row.agent.id}/${row.id}`);
```

Comment:
Please use the route types.

Reply:
Not sure what you mean, can you give an example?

Reply:
If this is what you mean, then it is fixed for all https://github.com/masumi-network/sokosumi/pull/248/commits/53024cd0834714636ddeac933615a3b0ec17fd9a